libs/coin-modules/coin-aptos/src/__tests__/api/index.integ.test.ts

```diff
@@ -79,4 +79,17 @@ describe("createApi", () => {
       expect(rawTx.max_gas_amount.toString()).toBe(DEFAULT_GAS.toString());
     });
   });
+
+  describe("listOperations", () => {
+    it("returns operations from account", async () => {
```

Comment:
You need to check operation attribute for IN and OUT transactions: hash, amount, recipient, sender, ...

pick one tx with funds moving out and another with funds entering

Reply:
addressed, please check if its ok now. thanks

Reply:
Attributes are not checked

Please see example : https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-modules/coin-tron/src/logic/listOperations.integ.test.ts#L186

Reply:
updated tests accordingly, thanks